lib/common/entropy_common.c

```diff
@@ -43,8 +43,14 @@ static U32 FSE_ctz(U32 val)
     assert(val != 0);
     {
 #   if defined(_MSC_VER)   /* Visual */
-        unsigned long r=0;
-        return _BitScanForward(&r, val) ? (unsigned)r : 0;
+        if (val != 0) {
```

Comment:
We've decided to no longer enforce separation between FSE & zstd. So this can just call `ZSTD_countTrailingZeros()` now.

So you can just `#include "zstd_internal.h"` and use that function.

Reply:
`FSE_ctz(U32 val)` is for `U32` type:
```c
U32 bitStream;
...
FSE_ctz(~bitStream | 0x80000000) >> 1;
```
While `ZSTD_countTrailingZeros(size_t val)` is for `size_t` type, there is an unnecessary cast in 64-bit build.

So the force-pushed last commit only fix `>> 3` bug.